<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomScanner</class>
 <widget class="QMainWindow" name="RoomScanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>717</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PCLViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Point Cloud</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QVTKWidget" name="qvtkWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>640</width>
            <height>480</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <property name="leftMargin">
           <number>50</number>
          </property>
          <property name="topMargin">
           <number>25</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Show Coordinate System</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_reset">
            <property name="text">
             <string>Reset Camera</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_stream">
            <property name="text">
             <string>Stream</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_save">
            <property name="text">
             <string>Save Frame</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_reg">
            <property name="text">
             <string>Registrate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_poly">
            <property name="text">
             <string>Polygonate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
       <zorder>qvtkWidget</zorder>
       <zorder></zorder>
       <zorder></zorder>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Polygonated mesh</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QVTKWidget" name="qvtkWidget_2" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>640</width>
            <height>480</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionLoad_Point_Cloud"/>
    <addaction name="actionShow_keypoints"/>
    <addaction name="actionShow_captured_frames"/>
    <addaction name="actionClear"/>
    <addaction name="actionSmooth_cloud"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionLoad_Point_Cloud">
   <property name="text">
    <string>Load Point Cloud</string>
   </property>
  </action>
  <action name="actionShow_keypoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show keypoints</string>
   </property>
  </action>
  <action name="actionShow_captured_frames">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show last captured frame</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionSmooth_cloud">
   <property name="text">
    <string>Smooth cloud</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton_poly</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>pushButton_reset</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
